docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<p>"Remove unnecessary gates" sounds very generic and overloaded in terms of what it means. Maybe we should be more specific like "Eject Single Qubit Operations" (or some other better name)? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>Agreed. Changed to "Eject single-qubit operations" here and in the outline at the start of the tutorial.</p>